invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/modal.overrides

```diff
@@ -18,8 +18,36 @@
 
 #community-selection-modal {
 
+  .ui.segment.community-list-container {
+    padding: 0;
+  }
+
   .community-list-results.content {
     height: calc(55vh - 10em);
+
+    .item {
+      padding: 2rem !important; //overriding semantic ui's !important
+
+      &.selected {
+        background-color: lighten(@borderColor, 65%);
+      }
+
+      .header {
+        a.ui.button.transparent {
+          color: @linkColor;
+        }
+      }
+
+      .extra.flex.width.auto {
+        display: flex;
```

Comment:
```.flex{
display: flex;
}
```
already defined in site.variables, adding it here is reduntant

Reply:
The `ui.items > .item .extra` is overriding the `.flex` class with `display: block`, so I had to specify it here.